data-prepper-api/src/main/java/org/opensearch/dataprepper/model/schemas/IfPresentAlsoRequire.java

```diff
@@ -0,0 +1,21 @@
+package org.opensearch.dataprepper.model.schemas;
```

Comment:
We should put this in the package `org.opensearch.dataprepper.model.annotations`.